EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 5190
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-4973
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155, 4973
+created: 2022-06-29
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP also extends account-bound token from [EIP-4973](https://eips.ethereum.org/EIPS/eip-4973), but the difference is that it can be transferred to other account for mobility between managing device.
```

Comment:
```suggestion
This EIP also extends account-bound token from [EIP-4973](./eip-4973), but the difference is that it can be transferred to other account for mobility between managing device.
```